library/Rules/DateTimeDiff.php

```diff
@@ -35,7 +35,7 @@ final class DateTimeDiff extends Standard
 
     private readonly Validatable $rule;
 
-    /** @param "years"|"months"|"days"|"hours"|"minutes"|"seconds"|"microseconds" $type */
+    /** @param string $type "years"|"months"|"days"|"hours"|"minutes"|"seconds"|"microseconds" */
```

Comment:
This was like that on purpose, it's a literal constant. See https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants